diff --git a/CMakeLists.txt b/CMakeLists.txt
--- a/CMakeLists.txt	(revision f3cf9ff73a5d323fa268ba8b155db0b8b02f7438)
+++ b/CMakeLists.txt	(date 1772121750023)
@@ -83,8 +83,10 @@
 message(STATUS "Build Fortran: ${FORTRAN}")
 option(CSHARP "Build CSharp interface" OFF)
 message(STATUS "Build CSharp: ${CSHARP}")
+option(JAVA "Build Java interface" OFF)
+message(STATUS "Build Java: ${JAVA}")
 
-if (FORTRAN OR CSHARP)
+if (FORTRAN OR CSHARP OR JAVA)
   set(BUILD_SHARED_LIBS ON)
 endif()
 
